tools/cldr-apps/js/src/views/DashboardPanel.vue

```diff
@@ -106,6 +110,34 @@ export default {
     this.fetchData();
   },
   methods: {
+    scrollto: function scrollto(event) {
+      const whence = event.target.getAttribute("notification");
```

Comment:
I suspect `scrollto: function` is superfluous

Reply:
Probably. Debugging an issue with "this".